dask_ml/_partial.py

```diff
@@ -171,20 +170,28 @@ def fit(model, x, y, compute=True, **kwargs):
     if y is not None:
         assert y.ndim == 1
         assert x.chunks[0] == y.chunks[0]
-    assert hasattr(model, 'partial_fit')
+    assert hasattr(model, "partial_fit")
     if len(x.chunks[1]) > 1:
         x = x.rechunk(chunks=(x.chunks[0], sum(x.chunks[1])))
 
     nblocks = len(x.chunks[0])
 
-    name = 'fit-' + dask.base.tokenize(model, x, y, kwargs)
+    name = "fit-" + dask.base.tokenize(model, x, y, kwargs)
     dsk = {(name, -1): model}
-    dsk.update({(name, i): (_partial_fit, (name, i - 1),
-                                          (x.name, i, 0),
-                                          (getattr(y, 'name', ''), i), kwargs)
-                for i in range(nblocks)})
+    dsk.update(
+        {
+            (name, i): (
+                _partial_fit,
+                (name, i - 1),
+                (x.name, i, 0),
+                (getattr(y, "name", ""), i),
+                kwargs,
+            )
+            for i in range(nblocks)
+        }
```

Comment:
I think I'm for black (it'd save me time), but this one-element-per-line is one formatting option I dislike, especially for many options.

Reply:
Black does have support for `# fmt: off` and `# fmt: on` flags to stop Black's formatting. I think we should use those if Black is used.

Reply:
Hmm... Agreed that this is less pleasant.

However, I'm inclined to do all or nothing here. Locally, I would set up [a pre-commit hook](https://github.com/ambv/black#version-control-integration) and then never worry about it again :)